onyxia-api/src/main/resources/schemas/ide/startupProbe.json

```diff
@@ -0,0 +1,16 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "startupProbe",
```

Comment:
You can use a title like 'Startup probe', there is no use for camel case here

Reply:
Fixed. Thanks!